src/main/java/org/galatea/starter/entrypoint/IexRestController.java

```diff
@@ -61,9 +62,23 @@
   @GetMapping(value = "${mvc.iex.getHistoricalPricesPath}", produces = {
       MediaType.APPLICATION_JSON_VALUE})
   public List<IexHistoricalPrices> getHistoricalPrices(
-      @RequestParam(value = "symbol") final String symbols, @RequestParam("range")
-      final String range, @RequestParam(value = "date", required = false) final String date) {
-    return iexService.getHistoricalPricesForSymbols(symbols,range,date);
+      @RequestParam(value = "symbol") final String symbols, @RequestParam(value = "range",
+      required = false) final String range, @RequestParam(value = "date", required = false)
+      final String date) {
+
+    if (symbols == null || symbols.length() == 0) {
```

Comment:
you do null or len=0 3 times. it is probably worth extracting to a utility method

Reply:
method added